mne_bids/path.py

```diff
@@ -405,6 +405,25 @@ def update(self, check=None, **entities):
         self._check()
         return self
 
+    def match(self):
+        """Get a list of all matching basenames.
```

Comment:
```suggestion
        """Get a list of all matching paths in the root directory.
```
Needs to be phrased better. @agramfort?